### ISSUE TITLE
Add support for 3 digit multiplications please

### ISSUE THREAD

**Issue by JgBr123** [AceLewis/my_first_calculator.py#192]
I am using this program in the main server of the company that i work on, but yesterday i realized it doesn't have 3 digit multiplications and i need it, if you could add it. It would be highly appreciated! BTW i starred your project, now i can make calculations in my programs.

**Comment by SwapnilSoni1999** [comment]
hmm good discovery lad

**Comment by theletter-zee** [comment]
I agree. This project is great due to the efficiency and speed of the program which is why I have recently brought it to the attention of my superiors but after I showed them the project I was swiftly told to leave their presence and I was almost at risk of losing my job but I think they would've been more impressed if 3 digit multiplication was added.
